### PR TITLE
Adjust inline svg techniques

### PR DESCRIPTION
**Pull request by sndrs**
_this is a much more pared down version of #7965, which wouldn't stop expanding and has been retired…_
### Changes to api
- don't prefix bespoke classes with `inline-`
- add first images to js (in `core.js`)
- add a js image lookup method, so we only add classes if we need to - means each image only needs be required once
- hide `inline-icon` svgs from non-svg capable browsers (IE8, android 2.*)
### Images
- inlines all header icons, including all instances of the batwing

http://www.browserstack.com/screenshots/54a2e03446d305adce32a13c8130ec6428278c5f

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/onward/breaking-news.js [hunk 1]
```diff
@@ -102,9 +105,13 @@ define([
 
                 if (alerts.length) {
                     $breakingNews = $breakingNews || bonzo(qwery('.js-breaking-news-placeholder'));
+                    marque36icon = svgs('marque36icon');
```
Comment (robertberry) [inline static/src/javascripts/projects/common/modules/onward/breaking-news.js]: Does this mean all the svgs are put in the JS?

File: static/src/javascripts/projects/common/views/svgs.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Include any images needed in templates here.
```
Comment (robertberry) [inline static/src/javascripts/projects/common/views/svgs.js]: Ignore my above comment ...

**Comment by robertberry** [comment]
:+1:

**Comment by sndrs** [comment]
will merge #8013 first so this works in IE8 et al

File: static/src/javascripts/projects/common/views/svgs.js (hunk 2 above)
Comment (sndrs) [inline static/src/javascripts/projects/common/views/svgs.js]: done

File: common/app/views/fragments/header.scala.html [hunk 3]
```diff
@@ -16,9 +16,7 @@
                     <a href="@Configuration.id.url/signin" data-link-name="User profile" data-toggle="popup--profile"
                        class="brand-bar__item--action popup__toggle" data-test-id="sign-in-link"
                        aria-haspopup="true">
-                        <span class="rounded-icon control__icon-wrapper">
-                            <i class="i i-profile-white-36"></i>
-                        </span>
+                        @fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"))
```
Comment (marchibbins) [inline common/app/views/fragments/header.scala.html]: Be sure to update the equivalent in `identityHeader` too.

Comment (sndrs) [inline common/app/views/fragments/header.scala.html]: nice one @marchibbins, thanks

Comment (marchibbins) [inline common/app/views/fragments/header.scala.html]: :+1:

**Pull request merged**